oxidation/libparsec/crates/protocol/schema/authenticated_cmds/pki_enrollment_list.json

```diff
@@ -0,0 +1,86 @@
+[
+    {
+        "label": "PkiEnrollmentList",
+        "major_versions": [
+            2,
+            3
+        ],
+        "introduced_in": "3.2",
+        "req": {
+            "cmd": "pki_enrollment_list"
+        },
+        "reps": {
+            "ok": {
+                "fields": {
+                    "enrollments": {
+                        "type": "List<PkiEnrollmentListItem>"
+                    }
+                }
+            }
+        },
+        "nested_types": {
+            "PkiEnrollmentListItem": {
+                "fields": {
+                    "enrollment_id": {
+                        "type": "UUID"
+                    },
+                    // Signature should be checked before loading.
+                    "submit_payload": {
+                        "type": "Bytes"
+                    },
+                    "submit_payload_signature": {
+                        "type": "Bytes"
+                    },
+                    "submitted_on": {
+                        "type": "DateTime"
+                    },
+                    "submitter_der_x509_certificate": {
+                        "type": "Bytes"
+                    }
+                }
+            }
+        }
+    },
+    {
+        "label": "PkiEnrollmentList",
+        "major_versions": [
+            2,
+            3
+        ],
+        "introduced_in": "2.8",
+        "req": {
+            "cmd": "pki_enrollment_list"
+        },
+        "reps": {
```

Comment:
From parsec `2.12.1` I can find in parsec/backend/pki.py
```python
    @api("pki_enrollment_list")
    @catch_protocol_errors
    async def api_pki_enrollment_list(
        self, client_ctx: AuthenticatedClientContext, msg: dict
    ) -> dict:
        if client_ctx.profile != UserProfile.ADMIN:
            return {
                "status": "not_allowed",
                "reason": f"User `{client_ctx.device_id.user_id}` is not admin",
            }
    ...
```
I don't see this kind of response here